training/lightning_trainer/trainingmodule.py

```diff
@@ -40,8 +40,9 @@ def finetune_function(self, pl_module: pl.LightningModule, epoch: int, optimizer
 
 class TransferTrainingModule(pl.LightningModule):
 
-    def __init__(self, learning_rate, num_target_classes):
+    def __init__(self, learning_rate, num_target_classes, model_arguments=None):
```

Comment:
`model_arguments={}` would avoid having to write additional if/else statements afterwards.

Reply:
No, don't do this! See the important warning here: https://docs.python.org/3/tutorial/controlflow.html#default-argument-values.  Having a mutuable as a default value will cause chaos

Reply:
I fail to see how it would affect `__init__`.
It also seems to be dead code.

Reply:
Good point. I didn't commit my line 46 since I had to change the pretrained path to run it outside docker. However, for this model_arguments part to do anything it should be changed to:

`self.aclp = AudioCLIP(pretrained=f'/app/assets/AudioCLIP-Full-Training.pt', **model_arguments)`